### PR TITLE
Tickets/instrm 447

### PR DESCRIPTION
**Pull request by yukimoritani**
I have assigned document IDs for Atsushi's documents. Taking this opportunity, the registry for IPMU documents have been made.

### CODE REVIEW

**Review by himorin — APPROVED** [review]

File: registries/ipmu/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+#### Durham
+
+To be assigned.
+
+#### Project Office
+
+| Number | Author |
+|:------:|:-------:|
+| 00 | Projcet office (unique) |
```
Comment (hassanxp) [inline registries/ipmu/README.md]: Typo: Project -> Project

File: registries/ipmu/documents.csv [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#document_id, title, author, url, date_submitted, comments
+"PFS-GEN-IPM000001-02", "NAOJ/Subaru-PFS so1ware assembly meeting", "Naoyuki Tamura",, "2018/06/05", "PFS-ics:01242/pfspo 23989"
```
Comment (hassanxp) [inline registries/ipmu/documents.csv]: so1ware? Is this a typo -> software?

File: registries/ipmu/documents.csv [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+"PFS-PFI-IPM001002-01", "Effects on Cobra operation from design and manufacture error", "Atsushi Shimono", "https://sumire.pbworks.com/w/file/107066907/PFS-SE-IPMU-00014-001-CobraOperation.pptx", "2016/04/13", "PFS-SE-IPMU-00014"
+"PFS-PFI-IPM001003-00", "AG conversion accuracy study", "Atsushi Shimono",, "2016/04/21", "PFS-SE-IPMU-00015/ pfspo 17886"
+"PFS-SPS-IPM001008-02", "SCR cabling design", "Atsushi Shimono", "https://github.com/Subaru-PFS/ics_doc/blob/master/SE-IPMU-00016.pptx", "2016/09/15", "PFS-SE-IPMU-00016"
+"PFS-SPS-IPM001009-01", "Exposure time control per camera (arm)", "Atsushi Shimono", "https://github.com/Subaru-PFS/ics_doc/blob/master/PFS-SE-IPMU-00017-001%2B-ExpTime.pptx", "2016/09/20", "PFS-SE-IPMU-00017. pfspo 18777"
+"PFS-ICS-IPM001002-01", "Data rate from reading out detectors", "Atsushi Shimono", "https://github.com/Subaru-PFS/ics_doc/tree/master/SSN-00001", "2012/08/28", "SSN-00001"
+"ICD-00070", "PFS ICS network connections (ICD 70)", "Atsushi Shimono", "https://github.com/Subaru-PFS/ics_doc/tree/master/SSN-00002", "2014/08/06", "SSN-00002"
```
Comment (hassanxp) [inline registries/ipmu/documents.csv:28]: Full doc code isn't present. Need to complete this?

Comment (yukimoritani) [reply registries/ipmu/documents.csv:28]: Well... ICDs (Interface Control Documents) are well-known as "ICD" and other ICDs don't have the code. So I didn't put the code fir this document.

**Pull request merged**